intercom_flutter/pubspec.yaml

```diff
@@ -1,15 +1,15 @@
 name: intercom_flutter
 description: Flutter plugin for Intercom integration. Provides in-app messaging
   and help-center Intercom services
-version: 7.8.4
+version: 7.8.5
 homepage: https://github.com/v3rm0n/intercom_flutter
 
 dependencies:
   flutter:
     sdk: flutter
   flutter_web_plugins:
     sdk: flutter
-  intercom_flutter_platform_interface: ^1.3.1
+  intercom_flutter_platform_interface: ^1.3.2
   intercom_flutter_web: ^0.3.2
```

Comment:
0.3.3 version is published so please update the same.

Reply:
Thanks. It's now done @deepak786!